zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/PopulationDataStore.scala

```diff
@@ -0,0 +1,24 @@
+package edu.uci.ics.cloudberry.zion.model.impl
+
+import edu.uci.ics.cloudberry.zion.model.schema._
+
+/**
+  * Created by nishadgurav on 1/19/17.
```

Comment:
my general suggestion is to remove the `created by author` template. since we are using git, that is easy to track.